{"uid":"b0efc26ed5f36f6b","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"9bbc2729137fa409b9bc8067987e1d0d","time":{"start":1707700151330,"stop":1707700161606,"duration":10276},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x565388117d93 <unknown>\n#1 0x565387dfb337 <unknown>\n#2 0x565387e46d60 <unknown>\n#3 0x565387e46e51 <unknown>\n#4 0x565387e8abc4 <unknown>\n#5 0x565387e6946d <unknown>\n#6 0x565387e8812d <unknown>\n#7 0x565387e691e3 <unknown>\n#8 0x565387e39135 <unknown>\n#9 0x565387e3a13e <unknown>\n#10 0x5653880dbe4b <unknown>\n#11 0x5653880dfdfa <unknown>\n#12 0x5653880c86d5 <unknown>\n#13 0x5653880e0a6f <unknown>\n#14 0x5653880ac69f <unknown>\n#15 0x565388105098 <unknown>\n#16 0x565388105262 <unknown>\n#17 0x565388116f34 <unknown>\n#18 0x7f8db8294ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f21379c6750>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...ct='Non Existing', page_title='My Workspace', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"6f2e72ad478decc03859beaadc9b7e28\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f2137898b80>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x565388117d93 <unknown>\nE       #1 0x565387dfb337 <unknown>\nE       #2 0x565387e46d60 <unknown>\nE       #3 0x565387e46e51 <unknown>\nE       #4 0x565387e8abc4 <unknown>\nE       #5 0x565387e6946d <unknown>\nE       #6 0x565387e8812d <unknown>\nE       #7 0x565387e691e3 <unknown>\nE       #8 0x565387e39135 <unknown>\nE       #9 0x565387e3a13e <unknown>\nE       #10 0x5653880dbe4b <unknown>\nE       #11 0x5653880dfdfa <unknown>\nE       #12 0x5653880c86d5 <unknown>\nE       #13 0x5653880e0a6f <unknown>\nE       #14 0x5653880ac69f <unknown>\nE       #15 0x565388105098 <unknown>\nE       #16 0x565388105262 <unknown>\nE       #17 0x565388116f34 <unknown>\nE       #18 0x7f8db8294ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1707700101783,"stop":1707700101783,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1707700101783,"stop":1707700101783,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1707700101784,"stop":1707700101784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x565388117d93 <unknown>\n#1 0x565387dfb337 <unknown>\n#2 0x565387e46d60 <unknown>\n#3 0x565387e46e51 <unknown>\n#4 0x565387e8abc4 <unknown>\n#5 0x565387e6946d <unknown>\n#6 0x565387e8812d <unknown>\n#7 0x565387e691e3 <unknown>\n#8 0x565387e39135 <unknown>\n#9 0x565387e3a13e <unknown>\n#10 0x5653880dbe4b <unknown>\n#11 0x5653880dfdfa <unknown>\n#12 0x5653880c86d5 <unknown>\n#13 0x5653880e0a6f <unknown>\n#14 0x5653880ac69f <unknown>\n#15 0x565388105098 <unknown>\n#16 0x565388105262 <unknown>\n#17 0x565388116f34 <unknown>\n#18 0x7f8db8294ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f21379c6750>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...ct='Non Existing', page_title='My Workspace', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"6f2e72ad478decc03859beaadc9b7e28\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f2137898b80>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x565388117d93 <unknown>\nE       #1 0x565387dfb337 <unknown>\nE       #2 0x565387e46d60 <unknown>\nE       #3 0x565387e46e51 <unknown>\nE       #4 0x565387e8abc4 <unknown>\nE       #5 0x565387e6946d <unknown>\nE       #6 0x565387e8812d <unknown>\nE       #7 0x565387e691e3 <unknown>\nE       #8 0x565387e39135 <unknown>\nE       #9 0x565387e3a13e <unknown>\nE       #10 0x5653880dbe4b <unknown>\nE       #11 0x5653880dfdfa <unknown>\nE       #12 0x5653880c86d5 <unknown>\nE       #13 0x5653880e0a6f <unknown>\nE       #14 0x5653880ac69f <unknown>\nE       #15 0x565388105098 <unknown>\nE       #16 0x565388105262 <unknown>\nE       #17 0x565388116f34 <unknown>\nE       #18 0x7f8db8294ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1707700151330,"stop":1707700161605,"duration":10275},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x565388117d93 <unknown>\n#1 0x565387dfb337 <unknown>\n#2 0x565387e46d60 <unknown>\n#3 0x565387e46e51 <unknown>\n#4 0x565387e8abc4 <unknown>\n#5 0x565387e6946d <unknown>\n#6 0x565387e8812d <unknown>\n#7 0x565387e691e3 <unknown>\n#8 0x565387e39135 <unknown>\n#9 0x565387e3a13e <unknown>\n#10 0x5653880dbe4b <unknown>\n#11 0x5653880dfdfa <unknown>\n#12 0x5653880c86d5 <unknown>\n#13 0x5653880e0a6f <unknown>\n#14 0x5653880ac69f <unknown>\n#15 0x565388105098 <unknown>\n#16 0x565388105262 <unknown>\n#17 0x565388116f34 <unknown>\n#18 0x7f8db8294ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"8bafb17c87bb1e73","name":"public ip address","source":"8bafb17c87bb1e73.txt","type":"text/plain","size":13},{"uid":"26443cd75fdbdd05","name":"Cookies","source":"26443cd75fdbdd05.json","type":"application/json","size":3871},{"uid":"36cd6ee8b89f916e","name":"Session Storage","source":"36cd6ee8b89f916e.json","type":"application/json","size":359},{"uid":"7662df36ff7171e3","name":"Local Storage","source":"7662df36ff7171e3.json","type":"application/json","size":208},{"uid":"9c1ce212b7873824","name":"Console Logs","source":"9c1ce212b7873824.json","type":"application/json","size":1521},{"uid":"7b4d928f0c4d8211","name":"Full Page Screenshot","source":"7b4d928f0c4d8211.png","type":"image/png","size":5706786},{"uid":"715c5965f7e24431","name":"URL","source":"715c5965f7e24431.uri","type":"text/uri-list","size":23},{"uid":"7069e887effb0d92","name":"Network Logs","source":"7069e887effb0d92.json","type":"application/json","size":23090}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"feature","value":"Login"},{"name":"epic","value":"Security"},{"name":"severity","value":"blocker"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az773-323"},{"name":"thread","value":"2315-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'elias@gmail.com'"},{"name":"password","value":"'12345Tr'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"b0efc26ed5f36f6b.json","parameterValues":["'elias@gmail.com'","'12345Tr'"]}